src/OrbitGl/TimeGraph.cpp

```diff
@@ -774,25 +752,18 @@ void TimeGraph::UpdateHorizontalSliderFromWorld() {
   slider_->SetPixelHeight(slider_width);
   slider_->SetNormalizedLength(static_cast<float>(width / time_span));
   slider_->SetNormalizedPosition(static_cast<float>(ratio));
-
-  slider_->SetOrthogonalSliderPixelHeight(vertical_slider_->GetVisible() ? slider_width : 0);
-  slider_->SetWidth(GetWidth() - (vertical_slider_->GetVisible() ? slider_width : 0));
 }
 
 void TimeGraph::UpdateVerticalSliderFromWorld() {
-  vertical_slider_->SetWidth(layout_.GetSliderWidth());
   float visible_tracks_height = GetTrackContainer()->GetVisibleTracksTotalHeight();
-  float max = std::max(0.f, visible_tracks_height - viewport_->GetWorldHeight());
+  float max = std::max(0.f, visible_tracks_height - vertical_slider_->GetHeight());
```

Comment:
Nit: Does this `max` refers to the max height of the vertical slider? How about rename it as `max_height`?

Reply:
yes!